server/cornerselling.lua

```diff
@@ -34,34 +45,34 @@ RegisterNetEvent('qb-drugs:server:giveStealItems', function(drugType, amount)
 end)
 
 RegisterNetEvent('qb-drugs:server:sellCornerDrugs', function(drugType, amount, price)
-    local src = source
-    local player = exports.qbx_core:GetPlayer(src)
-    local availableDrugs = getAvailableDrugs(src)
+    local player = exports.qbx_core:GetPlayer(source)
+    local availableDrugs = getAvailableDrugs(player.PlayerData.source)
 
     if not availableDrugs or not player then return end
 
     local item = availableDrugs[drugType].item
 
     local hasItem = player.Functions.GetItemByName(item)
     if hasItem.amount >= amount then
-        exports.qbx_core:Notify(src, locale('success.offer_accepted'), 'success')
-        exports.ox_inventory:RemoveItem(src, item, amount)
+        exports.qbx_core:Notify(player.PlayerData.source, locale('success.offer_accepted'), 'success')
+        exports.ox_inventory:RemoveItem(player.PlayerData.source, item, amount)
         player.Functions.AddMoney('cash', price, 'sold-cornerdrugs')
-        TriggerClientEvent('qb-drugs:client:refreshAvailableDrugs', src, getAvailableDrugs(src))
+        if config.policeCallChance >= math.random(1, 100) then
+            TriggerEvent('police:server:policeAlert', locale('info.possible_drug_dealing'))
```

Comment:
This event requires 3 args when called from the server: text, camId and playerSource
camId is optional, playerSource only when called from the client.
```suggestion
            TriggerEvent('police:server:policeAlert', locale('info.possible_drug_dealing'), nil, player.PlayerData.source)
```